From 56020a2c51720a058d52ed8ba5f094ff0ebd9916 Mon Sep 17 00:00:00 2001
From: Dominggoes Isakh <drjisakh@gmail.com>
Date: Sat, 24 Jan 2026 21:53:33 +0100
Subject: [PATCH 7/7] linker: Ignore library namespace

This hack will allow to access system libraries by vendor libraries.

Change-Id: Ic676f19627a1a6a8c39decf22d29e97660acba33
---
 linker/linker.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index 261ff2835..68aeaa7d6 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -1504,7 +1504,7 @@ static bool find_library_in_linked_namespace(const android_namespace_link_t& nam
     LD_LOG(kLogDlopen,
            "find_library_in_linked_namespace(ns=%s, task=%s): Not accessible (soname=%s)",
            ns->get_name(), task->get_name(), soname.c_str());
-    return false;
+    return true;
   }
 
   // if library is already loaded - return it
-- 
2.51.0

